fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneExecuteProperties.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * LuceneScanProperties.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.lucene;
+
+import com.apple.foundationdb.record.ExecuteProperties;
+import com.apple.foundationdb.record.metadata.expressions.KeyExpression;
+
+import javax.annotation.Nullable;
+import java.util.concurrent.ExecutorService;
+
+public class LuceneExecuteProperties extends ExecuteProperties {
```

Comment:
Maybe a javadoc could be helpful? I was told that a javadoc is required for a public class in Record Layer.